index.html

```diff
@@ -41,6 +41,155 @@ <h2>I'm a software developer</h2>
     </div>
     <img src="Media/image.svg" alt="" class="squares">
   </section>
+  <section class="grid-container recent-work">
+    <div class="title">
+      <h1>My Recent Works</h1>
+      <hr>
```

Comment:
- [ ] Its very nice implementing your header in an h1 tag, please make sure that your background design is same as the figma design in order to stick to the figma design. and also don't forget to follow the typo(font , color) and spacing(margin, padding).
### Your design👇
![recen](https://user-images.githubusercontent.com/66992901/198507449-828d0f89-67e0-4649-b233-bebc4a8760b6.PNG)
### figma design👇
![recfigma](https://user-images.githubusercontent.com/66992901/198507572-b2077459-2521-48a6-b0a3-153e67a7be62.PNG)